src/theme/css/noscript.css

```diff
@@ -0,0 +1,3 @@
+.left-buttons button {
+    display: none;
+}
```

Comment:
So, `noscript.css` is literally one CSS rule.

Is this change actually worth it? More specifically, do you expect to have a lot more JS-specific CSS than just this?

Reply:
Might not be worth it currently. I didn't look at the rest yet but I think a lot more things could be done without JS. Anyway, gonna remove this file and move the rule into the main CSS file with `body:not(.js)` and should be fine.